geniza/entities/templates/entities/person_detail.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'base.html' %}
+{% load static i18n render_bundle_csp fullurl %}
+
+{% block meta_title %}{{ page_title }}{% endblock meta_title %}
+{% block meta_description %}{{ page_description }}{% endblock meta_description %}
+{% block extrameta %}
+    {% spaceless %}
+        {% for lang_code in PUBLIC_SITE_LANGUAGES %}
+            {% language lang_code %}
+                <link rel="alternate" hreflang="{{ lang_code }}" href="{% fullurl "entities:person" person.pk %}" />
+            {% endlanguage %}
+        {% endfor %}
+    {% endspaceless %}
+{% endblock extrameta %}
+
+{% block main %}
+    <!-- person details -->
+    {% include "entities/snippets/person_header.html" %}
+    {# tabs #}
+    {% include "entities/snippets/person_tabs.html" %}
+    <div class="container">
+        <section class="metadata">
+            <h2 class="sr-only">
+                {# Translators: label for person metadata section #}
+                {% translate 'Metadata' %}
+            </h2>
+            {# metadata #}
+            <dl class="metadata-list primary">
+                {# Translators: label for a person's gender #}
+                <dt>{% translate 'Gender' %}</dt>
+                <dd>{{ person.get_gender_display }}</dd>
+                {# TODO: Add date range, autogenerated or override #}
+                {% if person.role %}
+                    <dt>
+                        {# Translators: Label for a person's social role #}
+                        {% translate 'Social role' %}
+                    </dt>
+                    <dd>{{ person.role }}</dd>
+                {% endif %}
+            </dl>
+            {# secondary metadata #}
+            <dl class="metadata-list secondary">
+                {% if person.names.count > 1 %}
+                    <dt>
+                        {# Translators: label for alternative names for a person #}
+                        {% blocktranslate count counter=person.names.count|add:-1 trimmed %}
```

Comment:
would it make sense to implement a custom queryset for non-primary names so you could easily filter without all the extra logic here in the template?